docker-vertica/Dockerfile

```diff
@@ -141,6 +141,7 @@ RUN set -x \
   && if [[ ($MINIMAL != "YES" && $MINIMAL != "yes") ]] ; then \
     apt-get install -y --no-install-recommends $JRE_PKG; \
   fi \
+  && apt-get -y upgrade \
```

Comment:
This adds about 20mb to the image though since old versions of packages that we upgraded are still in the image layers. This size will slowly keep creeping up. Can we try changing the `BASE_OS_VERSION`?  Perhaps to `focal-20230301`

Reply:
Don't we want to remove this? If we are building with the latest base image it shouldn't be necessary anymore.

Reply:
I think this is important.  The focal base image isn't kept perfectly up-to-date with all the packages it includes.  This upgrade will do that.  In my tests, this eliminated a number of low and medium severity security scan hits.

I found a way to do it without increasing the size of the image.  I just squash the layers into one after the upgrade.